### PR TITLE
Fix incorrect link in Wild Winds Stance

### PR DESCRIPTION
**Pull request by rectulo**
remove wrong link in Wild Wind Stances (Kingmaker)

### CODE REVIEW

File: packs/spells/wild-winds-stance.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
...
-            "value": "<p>You take on the stance of the flowing winds, sending out waves of energy at a distance. You can make wind crash unarmed Strikes as ranged Strikes against targets within 30 feet. These deal 1d6 bludgeoning damage; are in the brawling group; and have the agile, nonlethal, propulsive, and unarmed traits. Wind crash Strikes ignore @UUID[Compendium.pf2e.kingmaker-features.Item.Concealed]{Concealement} and all cover.</p>\n<p>While in <em>wild winds stance</em>, you gain a +2 circumstance bonus to AC against ranged attacks.</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Stance: Wild Winds Stance]</p>"
+            "value": "<p>You take on the stance of the flowing winds, sending out waves of energy at a distance. You can make wind crash unarmed Strikes as ranged Strikes against targets within 30 feet. These deal 1d6 bludgeoning damage; are in the brawling group; and have the agile, nonlethal, propulsive, and unarmed traits. Wind crash Strikes ignore Concealement and all cover.</p>\n<p>While in <em>wild winds stance</em>, you gain a +2 circumstance bonus to AC against ranged attacks.</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Stance: Wild Winds Stance]</p>"
```
Comment (simonward) [inline packs/spells/wild-winds-stance.json:14]:
```suggestion
            "value": "<p>You take on the stance of the flowing winds, sending out waves of energy at a distance. You can make wind crash unarmed Strikes as ranged Strikes against targets within 30 feet. These deal 1d6 bludgeoning damage; are in the brawling group; and have the agile, nonlethal, propulsive, and unarmed traits. Wind crash Strikes ignore concealement and all cover.</p>\n<p>While in <em>wild winds stance</em>, you gain a +2 circumstance bonus to AC against ranged attacks.</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Stance: Wild Winds Stance]</p>"
```

**Comment by TikaelSol** [comment]
How is this an incorrect link?

**Comment by rectulo** [comment]
It seems Concealed (from Kingmaker-features) was wrongly used there for Concealment..

**Comment by TikaelSol** [comment]
Ah. It should link to the concealed condition

File: packs/spells/wild-winds-stance.json (hunk 1 above)
Comment (TikaelSol) [inline packs/spells/wild-winds-stance.json:14]:
```suggestion
            "value": "<p>You take on the stance of the flowing winds, sending out waves of energy at a distance. You can make wind crash unarmed Strikes as ranged Strikes against targets within 30 feet. These deal 1d6 bludgeoning damage; are in the brawling group; and have the agile, nonlethal, propulsive, and unarmed traits. Wind crash Strikes ignore @UUID[Compendium.pf2e.conditionitems.Item.Concealed]{concealement} and all cover.</p>\n<p>While in <em>wild winds stance</em>, you gain a +2 circumstance bonus to AC against ranged attacks.</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Item.Stance: Wild Winds Stance]</p>"
```

**Pull request merged**